pkgs/by-name/wl/wlx-overlay-s/package.nix

```diff
@@ -58,6 +61,10 @@ rustPlatform.buildRustPackage rec {
 
   env.SHADERC_LIB_DIR = "${lib.getLib shaderc}/lib";
 
+  prePatch = ''
+    cp ${./Cargo.lock} Cargo.lock
```

Comment:
whats the issue with lockfile from src ?

Reply:
The tagged commit is https://github.com/galister/wlx-overlay-s/commit/0fd356f3062edc60a0c65469ac9b428ae5c9b6a6, but that one didn't update the lockfile with the new crate version. That's why I took the `Cargo.lock` from https://github.com/galister/wlx-overlay-s/commit/8ae65083f90b34232da615bdb3bf1e9aa72fe4e5